src/restart_windows.nim

```diff
@@ -0,0 +1,120 @@
+## Procedures necessary for implementing :restart on Windows
```

Comment:
It feels daft to say this now but we should probably explain that it restarts Firefox rather than Windows

Reply:
It says "re*start*", not "re*boot*" 😜

I'll try and think of something better, though. I wanna try and improve the documentation anyway.